integration/spark2/src/main/scala/org/apache/spark/sql/execution/strategy/CarbonLateDecodeStrategy.scala

```diff
@@ -673,4 +673,26 @@ private[sql] class CarbonLateDecodeStrategy extends SparkStrategy {
     supportCodegen && vectorizedReader.toBoolean &&
     cols.forall(_.dataType.isInstanceOf[AtomicType])
   }
+
+  private def createHadoopFSRelation(relation: LogicalRelation) = {
+    val sparkSession = relation.relation.sqlContext.sparkSession
+    relation.catalogTable match {
+      case Some(catalogTable) =>
+        HadoopFsRelation(new CatalogFileIndex(
+          sparkSession,
+          catalogTable, relation.relation.sizeInBytes),
+          catalogTable.partitionSchema,
+          catalogTable.schema,
+          catalogTable.bucketSpec,
+          new SparkCarbonTableFormat,
+          catalogTable.storage.properties)(sparkSession)
+      case _ =>
+        HadoopFsRelation(new InMemoryFileIndex(sparkSession, Seq.empty, Map.empty, None),
```

Comment:
move `new InMemoryFileIndex(sparkSession, Seq.empty, Map.empty, None)` to next line

Reply:
ok